dolphin-dnn/src/test/java/edu/snu/dolphin/dnn/NeuralNetworkTest.java

```diff
@@ -58,17 +58,17 @@
 
   private final Matrix input = matrixFactory.create(new float[]{77, 57, 30, 26, 75, 74, 87, 75});
   private final Matrix expectedOutput =
-      matrixFactory.create(new float[]{5.61329743164e-01f, 5.97883503916e-01f, 4.79822915984e-01f});
+      matrixFactory.create(new float[]{5.96001906923e-01f, 5.54388444438e-01f, 4.88766051614e-01f});
```

Comment:
I thought that only the orientation of matrices would be changed in the tests (row/column), but it seems that many values in the tests have been changed too. What's the reason?